logic/security.go

```diff
@@ -17,17 +17,17 @@ const (
 	// ALL_NETWORK_ACCESS - represents all networks
 	ALL_NETWORK_ACCESS = "THIS_USER_HAS_ALL"
 
-	master_uname     = "masteradministrator"
-	Unauthorized_Msg = "unauthorized"
-	Unauthorized_Err = models.Error(Unauthorized_Msg)
```

Comment:
why was unauthorized message removed altogether ?
ideally you should use `unauthorized` when the jwt authentication fails and `forbidden` should be used when authentication passes but user doesn't have enough permissions to access a resource...

https://github.com/gravitl/netmaker/blob/e9e20210129ab3463dacbd833d805f65a042d440/logic/security.go#L150 - on error here the code should be set to `401` rather than `403`

Reply:
Fixed.